### ISSUE TITLE
Missing headers from VCF output: ##reference, ##contig

### ISSUE THREAD

**Issue by theocole** [WMRGL/IdentityCheck#2]
In accordance with requirements from GEL, the following headers are missing from the VCF output from the xml_parser:
```
##reference=<name of the fasta file used>
##contig=<as per the specification>
```

**Comment by theocole** [comment]
The reference is based on a liftover carried out by @chipmash from gnomad SNPs (37) https://hgdownload.soe.ucsc.edu/goldenPath/hg38/bigZips/hg38.fa.gz

URL references to contigs can be found at https://hgdownload.soe.ucsc.edu/goldenPath/hg38/chromosomes/

**Comment by theocole** [comment]
fixed by 21947bdd10265f45f43f4c1d16403d71b60b47ce
